typedarrays.go

```diff
@@ -698,7 +837,11 @@ func (a *typedArrayObject) isValidIntegerIndex(idx int) bool {
 }
 
 func (a *typedArrayObject) _putIdx(idx int, v Value) {
-	v = v.ToNumber()
+	if a.typedArray.exportType() == typeBigInt64Array {
```

Comment:
Could that be replaced with `toNumeric()`?

Reply:
Setting a Number value when using BigInt64Array will throw an exception, so it cannot be replaced.

Reply:
Yes. Although it looks like a break of abstraction, the spec explicitly calls for it (https://tc39.es/ecma262/#sec-typedarraysetelement). And it can't even be moved to `typedArray.set()` because it will break evaluation order. Oh, well.. :man_shrugging: 

It think though that you can use a type assertion (or a type switch) here, directly checking `a.typedArray` against `*bigInt64Array` and `*bigUint64Array`. Should be faster than two virtual calls.